### PR TITLE
New package: spnavcfg-1.1-1

### PR DESCRIPTION
**Pull request by CtrlC-Root**
#### Testing the changes
- I tested the changes in this PR: **YES**

With the following:
* libspnav-1.1_1
* 3DConnexion SpaceMouse Pro Wireless

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64

### CODE REVIEW

**Comment by CtrlC-Root** [comment]
Hmm some of these architectures are failing because the `uic` binary provided by `qt5-host-tools` is missing. Which is odd because at least on x86_64 `qt5-devel` has a dependency on `qt5-host-tools`. Let me investigate this a bit.

**Comment by CtrlC-Root** [comment]
Actually I think this is a cross compilation issue. I can verify the `uic` binary is available (ex. `./masterdir/usr/aarch64-linux-gnu/usr/lib/qt5/bin/uic`) but I can't figure out how to either adjust the `PATH` environment variable to include it or detect it in the package template file to pass to the Makefile via the `UIC` environment variable.

File: srcpkgs/spnavcfg/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'spnavcfg'
+pkgname=spnavcfg
+version=1.1
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config qt5-host-tools"
+makedepends="libspnav-devel qt5-devel"
+depends="libspnav qt5-core qt5-gui qt5-widgets"
```
Comment (classabbyamp) [inline srcpkgs/spnavcfg/template:8]: some of these may be autodetected

File: srcpkgs/spnavcfg/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+short_desc="Spacenav interactive configuration GUI"
+maintainer="Alexandru Barbur <alex@ctrlc.name>"
+license="GPL-3.0-or-later"
+homepage="http://spacenav.sourceforge.net"
+changelog="https://github.com/FreeSpacenav/spnavcfg/releases"
+distfiles="https://github.com/FreeSpacenav/spnavcfg/releases/download/v${version}/${pkgname}-${version}.tar.gz"
```
Comment (classabbyamp) [inline srcpkgs/spnavcfg/template:14]:
```suggestion
distfiles="https://github.com/FreeSpacenav/spnavcfg/releases/download/v${version}/spnavcfg-${version}.tar.gz"
```

Comment (CtrlC-Root) [reply srcpkgs/spnavcfg/template]: Fixed in https://github.com/void-linux/void-packages/commit/e462fc35ebda39fe2cf10d63b5423f1d03d4a0a0. I went back and forth on this and found examples of packages which do both haha. I'm guessing the rule here is to only template the `version` part?

File: srcpkgs/spnavcfg/template (hunk 1 above)
Comment (CtrlC-Root) [reply srcpkgs/spnavcfg/template:8]: Waiting for feedback at https://github.com/void-linux/void-packages/pull/44959#discussion_r1259948835 and then I'll address this.

Comment (CtrlC-Root) [reply srcpkgs/spnavcfg/template]: Fixed in https://github.com/void-linux/void-packages/commit/a9e1cf978d63c9df85fd5014bef3ba18733af592.

**Pull request merged**